app/Platform/Actions/UpdatePlayerGameMetrics.php

```diff
@@ -90,6 +91,7 @@ public function execute(PlayerGame $playerGame, bool $silent = false): void
             'points_weighted_total' => $game->TotalTruePoints,
             'points_weighted' => $pointsWeighted,
             'created_at' => $createdAt,
+            'achievements_unlocked_softcore' => $achievementsUnlockedSoftcoreCount,
```

Comment:
recommend moving up so it's grouped with the other `achievements_unlocked*` fields.